pipeline-model-json-shaded/pom.xml

```diff
@@ -54,12 +54,34 @@
             <includes>
               <include>com.github.fge:*</include>
               <include>com.fasterxml*:*</include>
+              <include>joda-time:joda-time:*</include>
+              <include>com.google.guava:guava:*</include>
+              <include>com.googlecode.libphonenumber:*</include>
+              <include>org.mozilla:rhino:*</include>
```

Comment:
Yyyyup. But again, better safe than sorry. The freakin' `libphonenumber` ended up being a problem, so I just went for everything at that point.